crates/runtime/src/accelerated_table/refresh.rs

```diff
@@ -152,9 +154,26 @@ impl Refresher {
 
         let cache_provider = self.cache_provider.clone();
 
+        let refresh_check_interval = self.refresh.read().await.check_interval;
+
         tokio::spawn(async move {
+            // first refresh is on start, thus duration is 0
+            let mut next_scheduled_refresh_timer = Some(sleep(Duration::from_secs(0)));
+
             loop {
+                let scheduled_refresh_fut: BoxFuture<()> = match next_scheduled_refresh_timer.take()
```

Comment:
nit: just call it schedule_refresh_future.

Reply:
Updated, thank you